tutorials/continuous-aggs-tutorial.md

```diff
@@ -99,7 +99,7 @@ continuous aggregate query every 30 minutes.
 
 ```sql
 CREATE VIEW cagg_rides_view WITH
-(timescaledb.continuous, timescaledb.refresh_interval = ’30m’)
+(timescaledb.continuous, timescaledb.refresh_interval = '30m')
```

Comment:
Could you also add an example on how to create continuous aggregate views for hypertables that use "integer" based timestamps. Occasional question on support slack. (There is an example in tsl/test/sql/continuous_aggs.sql).  Either here or to the API docs,  whichever you think is more appropriate.

Reply:
I think it should be in the tutorial. I'll add it.

Reply:
From what I understand, integer-based timestamps are in microseconds. (Kind of fine-grained if you ask me. Easy to make a mistake and make the system unusable. Is there a request for microsecond `refresh_interval`?)

Reply:
Added just a paragraph about that in the tutorial.

Reply:
@mkindahl No, that's not the case. It can be anything the user wants to interpret it as, not necessarily microseconds. (The new paragraph should be removed.)  You will find an example with inetegr based timestamp and cont. aggregates here:  tsl/test/sql/continuous_aggs_bgw_drop_chunks.sql